app/views/posts/show.html.erb

```diff
@@ -1 +1,27 @@
-<h1>User posts with Id</h1>
\ No newline at end of file
+
+<% @posts.each do |post| %>
+<ul class="blog-container">
+ <li class="blog-card">
+      <a href="<%= user_post_path(post.author_id, post.id) %>" class="blog">
+        <h2><%= post.title %> by <%= User.find(post.author_id).name %></h2>
+        <p><%= post.text %></p>
+        <div class="comment-like-post">
+        <span>Comments: <%= post.comments.length %></span>
+        <span>Likes: <%= post.likes.length %></span>
+        </div>
+      </a>
+    </li>
+    </ul>
+
+    <ul class="comment-container">
+      <% unless post.comments.any? %>
+        <h2>No Comments</h2>
+      <% end %>
+      <% post.recent_comments.each do |comment| %>
+        <li >
+          <span><%= User.find(comment.user_id).name %>:</span>
+          <span><%= comment.text %></span>
+        </li>
+      <% end %>
+    </ul>
+  <% end %>
```

Comment:
Nice job so far. However, as requested by the previous reviewer, kindly make sure that this show page displays all the comments of a given post and not just the 5 recent comments. 

### Your design 

![Screenshot from 2023-09-28 22-13-39](https://github.com/manzitresor/blog_app/assets/65233860/9698aac4-a97a-4193-8231-c805123ac8c9)



### Given wireframe

![Screenshot from 2023-09-28 22-17-52](https://github.com/manzitresor/blog_app/assets/65233860/18f20daa-608c-49fc-bbbc-a92fbaf60755)